Gems/Atom/Feature/Common/Assets/Shaders/Materials/Silhouette/SilhouetteGather_MaterialSrg.azsli

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once 
+
+#include <Atom/Features/SrgSemantics.azsli>
+#include "../MaterialInputs/SilhouetteInput.azsli"
+
+ShaderResourceGroup MaterialSrg : SRG_PerMaterial
+{
+    float3 m_color;
+    float m_alpha;
+    float m_depthBias;
+    float m_outlineSize;
+    float m_padding0;
```

Comment:
Minor: Although not needed (as that is how it will work out under the hood) just make this float2 m_padding0 as then it will end exactly at 16 byte boundary in order to avoid any confusion.